komga-webui/public/index.html

```diff
@@ -6,6 +6,11 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <title>Komga</title>
     <link rel="icon" href="/favicon.ico" th:href="@{/favicon.ico}">
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" th:href="@{/apple-touch-icon.png}">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" th:href="@{/favicon-32x32.png}">
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" th:href="@{/favicon-16x16.png}">
+    <meta name="msapplication-TileColor" content="#08397f">
+    <meta name="msapplication-TileImage" content="/mstile-144x144.png">
```

Comment:
I'm surprised this works with a servlet context path though 🤔

Reply:
Fixed